underfs/pom.xml

```diff
@@ -30,6 +30,7 @@
     <module>s3a</module>
     <module>swift</module>
     <module>wasb</module>
+    <module>cos</module>
```

Comment:
put `<module>cos</module>` before `<module>gcs</module>`---sorted in text order

Reply:
OK